sdk/core/azure-core/azure/core/tracing/decorator.py

```diff
@@ -30,7 +30,7 @@
 from typing import Callable, Any, TypeVar, overload
 from typing_extensions import ParamSpec
 from .common import change_context, get_function_and_class_name
-from . import SpanKind
+from . import SpanKind as _SpanKind
```

Comment:
this seems breaking how are we documenting this to users?

Reply:
This was accidentally added in last PR :).